generated/nidcpower/nidcpower/session.py

```diff
@@ -4880,7 +4880,7 @@ def create_advanced_sequence(self, sequence_name, property_names, set_as_active_
         for prop in property_names:
             if prop not in Session.__base__.__dict__:
                 raise KeyError('{0} is not an property on the nidcpower.Session'.format(prop))
-            if not isinstance(Session.__base__.__dict__[prop], _attributes.Attribute) and not isinstance(Session.__base__.__dict__[prop], _attributes.AttributeEnum):
+            if not isinstance(Session.__base__.__dict__[prop], _attributes.Attribute) and not isinstance(Session.__base__.__dict__[prop], _attributes.AttributeEnum) and not isinstance(Session.__base__.__dict__[prop], _attributes.AttributeEnumWithConverter):
```

Comment:
Shreyas' suggestion I think would get rid of this, which to be honest is a bit of codesmell.

Reply:
Yes, I have updated it in my latest commit.